src/server/worldserver/Master.cpp

```diff
@@ -108,6 +108,43 @@ class FreezeDetectorRunnable : public ACORE::Runnable
     }
 };
 
+void Master::LodaModuleConfigSettings()
+{
+    Tokenizer configFileList(CONFIG_FILE_LIST, ',');
+    for each (std::string configFile in configFileList)
```

Comment:
I think you must remove "each" and that's it (I'm not really into C++ but I looked at other loops in the code and they don't use "each")

Reply:
Ok. This is a c++ feature. The compiler used by Travis/pr may not be compatible.

Reply:
https://docs.microsoft.com/fi-fi/cpp/dotnet/for-each-in

> for each, in
Iterates through an array or collection. This *non-standard* keyword is available in both C++/CLI and native C++ projects. However, its use is *not recommended*. Consider using a *standard* [Range-based for Statement (C++)](https://docs.microsoft.com/fi-fi/cpp/cpp/range-based-for-statement-cpp) instead.

Its a microsoft feature

Reply:
@Rochet2 Thank you for your answer! :smiley: